utils/exchange-bootstrap.sh

```diff
@@ -79,21 +99,17 @@ curl -sS --fail -u "${USERNAME}:${PASSWORD}" -X POST --header "Content-Type: app
 	"https://api.github.com/repos/${EXCHANGE_ORG}/${REPO_NAME}/keys"
 
 # GitHub: create a user-scope token
-echo "Github: Creating a Github user-scoped token"
-curl -sS --fail -u "${USERNAME}:${PASSWORD}" -X POST --header "Content-Type: application/json" \
-	-d '{"scopes": ["public_repo"], "note": "CircleCI: '"${REPO_NAME}"'"}' \
-	"https://api.github.com/authorizations" | jq ".token" > "/tmp/${PACK}_user_token"
-
+echo -n "${GITHUB_PACK_PAT}" > "/tmp/${PACK}_user_token"
```

Comment:
Drop writing this to a file. When it's already in a var just use it from the var.